src/util.h

```diff
@@ -36,6 +36,8 @@ typedef enum {
     LD_STR_HEX       /* %La */
 } ld2string_mode;
 
+int prefixmatch(const char *pattern, int patternLen, 
+                   const char *prefixStr, int prefixStrLen, int nocase);
```

Comment:
The indentation seems incorrect. Don't we have the clang format? Or does it look odd just here?